react/src/components/atoms/forms/InputSync/index.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import is from 'is';
+import deepEqual from 'fast-deep-equal';
+import { FormContext } from '../Input/context';
+
+class InputSync extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: {},
+      syncContent: this.syncContent,
+      children: this.props.children,
+      formIds: this.props.formIds
+    };
+  }
+  componentDidMount() {
+    this.checkFormContext();
+  }
+  checkFormContext = () => {
```

Comment:
Can you add comment about what checkFormContext is doing, and also in Input there's also checkFormContext but they are doing different things.